app/server.js

```diff
@@ -15,44 +46,25 @@ const { logger } = require('./logger');
     }
   });
 
-  let intervalId;
-  let app;
-  const pollDuration = 1000;
+  const appType = await determineAppTypeToStart({
+    canStartApp: false, appType: undefined, attempt: 1, pollDuration: 1000,
```

Comment:
you don't need `appType: undefined` as it will be undefined anyway

Reply:
gives me a lint error when I don't pass anything in

Reply:
put default into the function?

Reply:
I don't see why you can't set the appType before it goes into the function?

Reply:
still doesn't like that. Is it perhaps due to it being an object that is passed in